clusterloader2/pkg/provider/util.go

```diff
@@ -22,6 +22,10 @@ import (
 	sshutil "k8s.io/kubernetes/test/e2e/framework/ssh"
 )
 
+const (
+	noManagedPrometheus = "no managed Prometheus service for this cloud provider"
```

Comment:
Nit: Please use ErrNoManagedPrometheus = errors.New("no managed Prometheus service for this cloud provider")

This way it will be possible to check error kind without string matching.

Can be done as follow up PR

Reply:
Done.